### PR TITLE
fix randomness on keystore

### PR DESCRIPTION
**Pull request by fedekunze**
**Description:**

Thank you! 🚀

---

For contributor:

- [ ] Added entries in `PENDING.md` with issue # and GitHub username
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

**Comment by jbibla** [comment]
nice! does this actually _fix_ the issue?

File: test/unit/specs/scripts/keystore.spec.js [hunk 1]
```diff
@@ -7,6 +7,9 @@ import {
   getKey
 } from "renderer/scripts/keystore.js"
 
+// to test Web API Crypto
+import crypto from "crypto"
```
Comment (faboweb) [inline test/unit/specs/scripts/keystore.spec.js]: nice

**Comment by faboweb** [comment]
LGTM
did you test the account creation with this?

**Comment by fedekunze** [comment]
> did you test the account creation with this?

yes, works fine

**Pull request merged**